lib/Runtime/Library/WasmLibrary.cpp

```diff
@@ -307,6 +307,42 @@ namespace Js
 
     void WasmLibrary::WasmLoadIndirectFunctionTables(Wasm::WasmModule * wasmModule, ScriptContext* ctx, Var** indirectFunctionTables, Var* localModuleFunctions, Var* importFunctions)
     {
+        ArenaAllocator alloc(_u("WasmTempIndirectTable"), ctx->GetThreadContext()->GetPageAllocator(), Js::Throw::OutOfMemory);
```

Comment:
I decided to implement it and I think in the end it's better because we don't have to fill additional spaces in the table.
When dealing with indirect table (call or filling it) we have to map the signature id first, that way we always deal with only a set of unique signatures